compact/hcal/hcal_barrel_gdml.xml

```diff
@@ -0,0 +1,7682 @@
+        <position name="OuterHCalTile05_EJ200_v108" unit="mm" x="488.6627" y="803.9563" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v109" unit="mm" x="488.7138" y="804.3445" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v110" unit="mm" x="488.7138" y="804.3445" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v111" unit="mm" x="488.6627" y="804.7327" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v112" unit="mm" x="488.6627" y="804.7327" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v113" unit="mm" x="488.5128" y="805.0945" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v114" unit="mm" x="488.5128" y="805.0945" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v115" unit="mm" x="488.2745" y="805.4052" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v116" unit="mm" x="488.2745" y="805.4052" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v117" unit="mm" x="487.9638" y="805.6435" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v118" unit="mm" x="487.9638" y="805.6435" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v119" unit="mm" x="487.602" y="805.7934" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v120" unit="mm" x="487.602" y="805.7934" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v121" unit="mm" x="485.2972" y="798.9898" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v122" unit="mm" x="485.2972" y="798.9898" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v123" unit="mm" x="485.5992" y="798.754" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v124" unit="mm" x="485.5992" y="798.754" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v125" unit="mm" x="485.9511" y="798.6024" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v126" unit="mm" x="485.9511" y="798.6024" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v127" unit="mm" x="486.33" y="798.5449" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v128" unit="mm" x="486.33" y="798.5449" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v129" unit="mm" x="486.7111" y="798.5853" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v130" unit="mm" x="486.7111" y="798.5853" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v131" unit="mm" x="487.0695" y="798.7209" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v132" unit="mm" x="487.0695" y="798.7209" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v133" unit="mm" x="487.3818" y="798.9429" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v134" unit="mm" x="487.3818" y="798.9429" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200_v135" unit="mm" x="487.6277" y="799.2367" z="1.5"/>
+        <position name="OuterHCalTile05_EJ200_v136" unit="mm" x="487.6277" y="799.2367" z="7.0"/>
+        <position name="OuterHCalTile05_EJ200
```

Comment:
Remove emtpy plugins environment.

Reply:
Not sure I understand this comment? What do you want removed?

Reply:
Ah - found it, done.

Reply:
My browser was struggling with the long file, so I think it misplaced the comment...